pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go

```diff
@@ -36,10 +36,10 @@ type To struct {
 	Hostnames []string `json:"hostnames,omitempty"`
 	// TargetRef is a reference to the resource that represents a group of
 	// request destinations.
-	TargetRef common_api.TargetRef `json:"targetRef,omitempty"`
+	TargetRef common_api.TargetRef `json:"targetRef"`
 	// Rules contains the routing rules applies to a combination of top-level
 	// targetRef and the targetRef in this entry.
-	Rules []Rule `json:"rules,omitempty"`
+	Rules *[]Rule `json:"rules,omitempty"`
```

Comment:
should be probably required as there is no point in creating a `to` item without `rules`